### PR TITLE
Forked repo and fixed auth

### PR DESCRIPTION
**Pull request by theDanielK**
- Forked Repo: https://github.com/jilow/root-nodejs
- Fixed Auth

### CODE REVIEW

File: build/src/root/utils/client.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            method,
+            url: `${this.apiPath}/${entity}`,
+            auth: this.apiAuth,
+            params: method === 'GET' ? data : null,
+            data: method !== 'GET' ? data : null,
+            headers: { 'Content-Type': 'application/json', 'Accept': 'application/json' },
+        };
+    }
+    parseResponse(res) {
+        switch (res.status) {
```
Comment (ncthbrt) [inline build/src/root/utils/client.js:80]: Just a quick note. Our success response codes aren't just 200. We also return 201 created for example. It'd be better here to check if successful using  `(200/100 | 0) === 2` or logical equivalent

File: build/src/root/insurance/application.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+"use strict";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```
Comment (ncthbrt) [inline build/src/root/insurance/application.js:2]: Not 100% familar with these sort of idioms. Are these functions and `Object.defineProperty(exports, "__esModule", { value: true });` for backwards compatability?

Comment (ncthbrt) [reply build/src/root/insurance/application.js:2]: Ah. See you've got typescript in the project and this is the compiled output. Is it typical to check the compiled output in?

**Comment by ncthbrt** [comment]
Thanks for the PR @theDanielK, happy to merge this once the status code issue is fixed.

**Comment by theDanielK** [comment]
Hey @ncthbrt, I haven't changed anything from the original project besides fixing the auth issue.
I can fix that, but perhaps that's best to do in a separate PR.

This at least makes it usable instead of perfect.

**Pull request merged**